storage_service/locations/models/package.py

```diff
@@ -1196,14 +1191,22 @@ def create_new_pointer_file_with_replication(
         replication_relationship = _get_replication_derivation_relationship(
             replica_package.uuid, replication_event_uuid
         )
-        new_relationships = old_premis_object.findall("relationship")
-        new_relationships += (replication_relationship,)
-        new_premis_object = premisrw.PREMISObject(
-            xsi_type=old_premis_object.xsi_type,
-            object_identifier=old_premis_object.find("object_identifier"),
-            object_characteristics=old_premis_object.find("object_characteristics"),
-            relationship=new_relationships,
-        )
+        new_relationships = [
```

Comment:
@replaceafill this change (using the data attr of the old relationship) should fix the `AttributeError: Instance of <class 'abc.PREMISRelationshipElement'> has no attribute split.` issue — what is happening is using kwargs to construct a PREMISObject doesn't allow multiple values in metsrw right now, unfortunately.

Sorry, I forgot to note that anywhere — it would have been helpful to do so.

Reply:
Thank you @cole. It's indeed fixed by this.